src/modules/CommandList.js

```diff
@@ -186,6 +186,21 @@ Commands.list = {
             }
         }
     },
+    kickbots: function(gameServer) {
+        for (var i = 0; i < gameServer.clients.length;) {
+            if (typeof gameServer.clients[i].remoteAddress == 'undefined') {
+                var client = gameServer.clients[i].playerTracker;
+                var len = client.cells.length;
+                for (var j = 0; j < len; j++) {
```

Comment:
This `for` loop isn't required, `FakeSocket.js` has a function `close` which removes player cells.